cmd/tether/attach.go

```diff
@@ -328,3 +364,11 @@ func (t *attachServerSSH) channelMux(in <-chan *ssh.Request, process *os.Process
 
 	detach()
 }
+
```

Comment:
Why we (and everyone else) keep copying this struct? I know it's just 4 lines and not going to change but it's just keep coming :) Docker one seems to have an exported version so maybe we should use it.

```
[vagrant@devbox:/opt/go/src/github.com/vmware/vic(fix)] grep winsize | grep struct
cmd/tether/tether_linux.go:324:type winsize struct {
vendor/github.com/docker/docker/pkg/term/term.go:27:type Winsize struct {
vendor/github.com/docker/docker/pkg/term/term_windows.go:23:type Winsize struct {
vendor/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/docker/pkg/term/term.go:27:type Winsize struct {
vendor/github.com/docker/libnetwork/Godeps/_workspace/src/github.com/docker/docker/pkg/term/term_windows.go:24:type Winsize struct {
vendor/github.com/go-swagger/go-swagger/vendor/github.com/jessevdk/go-flags/termsize.go:10:type winsize struct {
vendor/github.com/jessevdk/go-flags/termsize.go:10:type winsize struct {
vendor/github.com/kr/pty/util.go:17:type winsize struct {
```